app/src/main/java/org/thunderdog/challegram/ui/ProfileController.java

```diff
@@ -3839,7 +3839,7 @@ private void buildEditCells () {
     if (alwaysVisible || tdlib.suggestConvertToBroadcastGroup(chat.id)) {
       items.add(new ListItem(added ? ListItem.TYPE_SEPARATOR_FULL : ListItem.TYPE_SHADOW_TOP));
       items.add(new ListItem(ListItem.TYPE_SETTING, R.id.btn_convertToBroadcastGroup, 0, R.string.ConvertToBroadcastGroup));
-      items.add(new ListItem(ListItem.TYPE_DESCRIPTION, 0, 0, R.string.ConvertToBroadcastGroupDesc));
+      items.add(new ListItem(ListItem.TYPE_DESCRIPTION, 0, 0, Lang.plural(R.string.ConvertToBroadcastGroupDesc, tdlib.supergroupMaxSize())));
```

Comment:
This will cause crash, as plural isn't properly defined. See https://github.com/TGX-Android/Telegram-X/blob/main/docs/GUIDE.md#22-plurals

Reply:
ok, maybe done 3c33343